### ISSUE TITLE
Cannot filter out package on "Packages" page

### ISSUE THREAD

**Issue by yuxisun1217** [RedHatInsights/image-builder-frontend#242]
Hi there,

I try to add additional packages to my image but cannot filter out my packages.

![Screenshot from 2021-06-17 16-29-01](https://user-images.githubusercontent.com/19280786/122362983-28645580-cf8b-11eb-825e-a19c84a43f78.png)

From Ondrej:
```
Uncaught (in promise) Error: Request failed with status code 400
    exports createError.js:16
    exports settle.js:17
    onreadystatechange xhr.js:62
 createError.js:16:14
```

**Comment by jkozol** [comment]
@yuxisun1217 were you able to replicate this on `main` or just on `cloud.stage.redhat.com`

**Comment by jkozol** [comment]
@yuxisun1217 This is fixed now thanks to osbuild/image-builder#203. Can you verify this works for you and close this issue?

**Comment by yuxisun1217** [comment]
Thanks @jkozol ! Verified on https://cloud.stage.redhat.com/beta/insights/image-builder/imagewizard . Can filter out packages. Close this issue.
